src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,268 @@ must be specified using either:
+  Alternatively, you could set the full name and path of the executables of the desired compilers if you don't want MacPorts settings to interfere with your spack settings.
+* You should set the spec to be equal to the actual version of the compiler you installed.
+
+In ``packages.yaml``:
+
+.. code-block:: yaml
+
+   packages:
+   all:
+      compiler: [clang, gcc]
+      providers:
+         blas: [netlib-lapack]
+         lapack: [netlib-lapack]
+         mpi: [openmpi]
+
+   mpi:
+      buildable: false
+   openmpi:
+      externals:
+      - spec: openmpi@4.1.4
+         prefix: /opt/local
+   
+   netlib-lapack:
+      buildable: false
+      externals:
+      - spec: netlib-lapack@3.10.1
+         prefix: /opt/local
+   autoconf:
+      buildable: false
+      externals:
+      - spec: autoconf@2.71
+         prefix: /opt/local
+   automake:
+      buildable: false
+      externals:
+      - spec: automake@1.16.5
+         prefix: /opt/local
+   bzip2:
+      buildable: false
+      externals:
+      - spec: bzip2@1.0.8
+         prefix: /opt/local
+   gettext:
+      buildable: false
+      externals:
+      - spec: gettext@0.21
+         prefix: /opt/local
+   graphviz:
+      buildable: false
+      externals:
+      - spec: graphviz@2.50.0
+         prefix: /opt/local
+   libtool:
+      buildable: false
+      externals:
+      - spec: libtool@2.4.6
+         prefix: /opt/local
+   libx11:
+      buildable: false
+      externals:
+      - spec: libx11@1.8.1
+         prefix: /opt/local
+   m4:
+      buildable: false
+      externals:
+      - spec: m4@1.4.6
+         prefix: /usr
+   perl:
+      buildable: false
+      externals:
+      - spec: perl@v5.34.1
+         prefix: /opt/local
+   pkg-config:
+      buildable: false
+      externals:
+      - spec: pkg-config@0.29.2
+         prefix: /opt/local
+   tar:
+      buildable: false
+      externals:
+      - spec: tar@3.3.2
+         prefix: /usr
+   readline:
+      buildable: false
+      externals:
+      - spec: readline@8.1.2.000
+         prefix: /opt/local
+   unzip:
+      buildable: false
+      externals:
+      - spec: unzip@6.0
+         prefix: /usr
+   zlib:
+      buildable: false
+      externals:
+      - spec: zlib@1.2.12
+         prefix: /opt/local
+
+Notes:
+
+* OpenGL is not supported on modern Macs, hence those packages are not specified as they are on other platforms. 
+  This means that the optional ``glvis`` spec can't be built.
+* The version specs should be set to the actual versions of the packages you have, which will not neccesarily be the same as the above.
+
+The above spack settings and MacPorts packages will cover the basic installation of Serac. 
+If you want to build the optional devtools, you should install the additional packages with MacPorts:
+
+* cmake (CMake is present by default on current MacOS distributions, but is often out of date, hence we recommend installing this package.)
```

Comment:
This is required by Serac build and probably shouldn't be in this list.  It is also listed in the original list of required packages.